WordPress/gutenberg
# Font Library: add wp_font_face post type and scaffold font face REST API controller

## What?

- Adds a `wp_font_face` custom post type, which will be used to store installed font faces
- Scaffolds a `WP_REST_Font_Faces_Controller` with basic functionality, which we can build upon in subsequent PRs

The following routes are implemented:

```
GET wp/v2/font-families/<id>/font-faces
GET wp/v2/font-families/<id>/font-faces/<id>
POST wp/v2/font-families/<id>/font-faces
DELETE wp/v2/font-families/<id>/font-faces/<id>
```

## Why?

This is a step toward [refactoring the Font Library API endpoints](https://github.com/WordPress/gutenberg/issues/55278) to implement [the proposed design](https://github.com/WordPress/gutenberg/issues/57616).

## Testing Instructions

- Check that you can access all of the routes as a an administrator user to read, create, and delete font face posts

### Testing Instructions for Keyboard

n/a
